bot.ml

```diff
@@ -839,10 +839,21 @@ let callback _conn req body =
                         ~number:pr.issue.number
                       >>= function
                       | Ok reviews_info -> (
-                          if
-                            List.exists reviews_info.last_comments ~f:(fun c ->
-                                String.equal comment_info.id c.id
-                                && c.created_by_email)
+                          let comment =
+                            List.find reviews_info.last_comments ~f:(fun c ->
+                                String.equal comment_info.id c.id)
+                          in
+                          if Option.is_none comment then
```

Comment:
Did this instead of pattern matching in order to avoid one more level of indentation.

Reply:
Looks good!